client/my-sites/stats/pages/realtime/index.jsx

```diff
@@ -14,7 +14,6 @@ import StatsModuleTopPosts from 'calypso/my-sites/stats/features/modules/stats-t
 import { getMomentSiteZone } from 'calypso/my-sites/stats/hooks/use-moment-site-zone';
 import { requestSiteStats } from 'calypso/state/stats/lists/actions';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
-import AnnualHighlightsSection from '../../sections/annual-highlights-section';
```

Comment:
Are we going to use this one? If not, perhaps it could be a good cleanup opportunity.

Reply:
The `AnnualHighlightsSection` component is still in use on the Insights page.